src/app/models/types/portraitVideoCarousel.ts

```diff
@@ -34,4 +34,11 @@ export interface PortraitVideoItem {
       contentId?: string;
     };
   };
+  onClick?: () => void;
+}
+
+export interface NavigationButtonsProp {
+  scroll: (direction: 'left' | 'right') => void;
```

Comment:
We have a `Direction` type somewhere
```suggestion
  scroll: (direction: Direction) => void;
```